nf_core/bump_version.py

```diff
@@ -159,7 +159,13 @@ def bump_nextflow_version(pipeline_obj, new_version):
             (
                 r"nextflow-%E2%89%A5{}-brightgreen.svg".format(current_version.replace(".", r"\.")),
                 "nextflow-%E2%89%A5{}-brightgreen.svg".format(new_version),
-            )
+            ),
+            (
+                r"1.\s*Install\s*\[`nextflow`\]\(https://nf-co.re/usage/installation\)\s*\(`>={}`\)".format(
```

Comment:
```suggestion
                r"1.\s*Install\s*\[`nextflow`\]\(https://nf-co\.re/usage/installation\)\s*\(`>={}`\)".format(
```

Reply:
this is actually taken care of by the `replace` function in the end

Reply:
I don't think it is? The replace is only for the version which is inserted into the `{}` right?

Reply:
Aah okay I did read it wrong, but then we also need to escape the dot after the one - thanks!

Reply:
Yup! Didn't see that 1 😅